.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[source.crates-io]
+replace-with = "vendored-sources"
+
+[source.vendored-sources]
+directory = "vendor"
```

Comment:
this is going to use the vendored sourcesfor everyt build and we don't want that

Reply:
Well that's not actually true, the `%cargo_prep` macro overwrites it. Only when `%cargo_prep -V 0` or `%cargo_prep -v vendor`  the build will use vendored sources.